setup.py

```diff
@@ -33,7 +33,7 @@ def merge_req_lists(req_lists: Iterable[List[str]]) -> List[str]:
 ]
 
 sqlite_dependencies = [
-    "sqlalchemy>=1.4.27",
+    "sqlalchemy==1.4.27",
```

Comment:
It's better not to limit version to a specific one for no reason. There could be important updates in versions between `1.4.27` and `2.0`.
```suggestion
    "sqlalchemy>=1.4.27, <2.0",
```